Backpack/Map/Classes/BPKMapAnnotationView.m

```diff
@@ -0,0 +1,162 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKMapAnnotationView.h"
+
+#import <Backpack/Color.h>
+#import <Backpack/Spacing.h>
+
+#import "BPKMapAnnotationViewCalloutView.h"
+#import "BPKMapAnnotation.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface BPKMapAnnotationView()
+@property(nonatomic, nullable) BPKMapAnnotation *bpk_annotation;
```

Comment:
This name is a bit surprising, snake case isn't a convention we use outside of category added method names, but I supposes this is similar. 

This should be `readonly` no?